src/modules/tests/communication-preferences/communication-preferences.selector.ts

```diff
@@ -5,4 +5,5 @@ export const getCommunicationPreferenceUpdatedEmail
     communicationPreferences.updatedEmail ? communicationPreferences.updatedEmail : '';
 
 export const getCommunicationPreferenceType
-  = (communicationPreferences: CommunicationPreferences) => communicationPreferences.communicationMethod;
+  = (communicationPreferences: CommunicationPreferences) =>
+  communicationPreferences.communicationMethod ? communicationPreferences.communicationMethod : '';
```

Comment:
Would be a good idea to use `lodash.get` with the default value of an empty string